src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx

```diff
@@ -263,18 +263,24 @@ export const EVMDestinationAdaptorProvider = ({
   useEffect(() => {
     if (transactionStatus === "Transfer Completed") {
       if (!destinationBridge || transferTxHash) return;
+      const startTime = performance.now();
```

Comment:
Let's implement it once inside the `getTransferTxHashByNonce`?

Reply:
This time will be required  for Google Analytics

Reply:
You can use Google Analytics in method?

Reply:
We can't  because  there are 2 independent services:
1. Google Analytics
2. Helpers which later I plan to transfer into web3 integration service and it shouldn't know anything about Google Analytics. All operations should happen inside the adaptor.